canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -47,6 +47,10 @@ class PropertiesMain extends React.Component {
 	constructor(props) {
 		super(props);
 		this.propertiesController = new PropertiesController();
+		// Persist editorSize when resize() is not called
+		if (this.propertiesController.getInitialEditorSize() !== this.props.propertiesInfo.initialEditorSize) {
```

Comment:
This condition is required to persist editorSize when resize() is not called. 
If I remove this condition and go through following steps, editorSize is not persisted -
1. I open `list_paramDef.json` and click on `Resize` button --> Size is updated to Medium
2. Then I open `numberfield_paramDef.json` --> Opening size is Medium (Persisted editorSize from step 1). 
3. I don't click on `Resize` button in step 2
4. I open `list_paramDef.json` --> Opening size is Small which is default size (editorSize is not persisted)

![persist-editor-size](https://user-images.githubusercontent.com/25124000/108432019-9163d880-71f8-11eb-9de2-790dc1f837eb.gif)

Reply:
Why not check if `this.props.propertiesInfo.initialEditorSize` is set and set it?  Not sure why we need to check the old value.